test-fixtures/bundle-project/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```

Comment:
what's the reason for the new fixtures? Are they logically part of the TS conversion?